MonoGame.Framework/Graphics/SpriteBatcher.cs

```diff
@@ -108,12 +123,9 @@ public SpriteBatcher (GraphicsDevice device)
         /// <returns></returns>
         public SpriteBatchItem CreateBatchItem()
         {
-            SpriteBatchItem item;
-            if (_freeBatchItemQueue.Count > 0)
-                item = _freeBatchItemQueue.Dequeue();
-            else
-                item = new SpriteBatchItem();
-            _batchItemList.Add(item);
+            if (_batchItemCount >= _batchItemList.Count)
+                _batchItemList.Add(new SpriteBatchItem());
+            var item = _batchItemList[_batchItemCount++];

```

Comment:
We should probably grow the batch items better here.  Like don't add one more item... that ensures every quad causes at least one new allocation.  Instead maybe we can do something like this:
```
if (_batchItemCount >= _batchItemList.Count) 
{
   do
  {
     _batchItemList.Add(new SpriteBatchItem());
  } while (batchItemList.Count < batchItemList.Capacity)
}
```
So basically we follow whatever heuristic `List<>` observes when growing itself to add new batch items.